vcpkg/get_started/get-started-linux.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Install and use packages on Linux
+description: Tutorial guides the user through the process of installing and using packages on Linux with vcpkg.
+author: JavierMatosD
+ms.author: javiermat
+ms.topic: tutorial
+ms.date: 08/23/2023
+ms.prod: vcpkg
+#CustomerIntent: As a beginner C++ developer, I want to learn how to install and manage packages on a Linux environment using vcpkg, so that I can easily set up and maintain C++ projects with necessary dependencies.
+---
+
+# Tutorial: Install and use packages on Linux with vcpkg
+
+This tutorial shows how to set up a C++ "Hello World" CMake project using the `fmt` library. Manage dependencies, construct the application, and execute the project with the help of vcpkg. This guide is tailored for a Linux environment.
+
+## Prerequisites
+
+The following is required for this tutorial:
+
+- A terminal to work in.
+- `git`, `CMake`, and a C++ compiler installed on your system
+- vcpkg installed on your system
+
+## 1 - Set up the project
```

Comment:
```suggestion
## Set up the project
```

Tutorial template says not to use numbers in steps

Reply:
https://review.learn.microsoft.com/en-us/help/patterns/level4/article-tutorial?branch=main#use-h2s-to-describe-tasks

It says to use number in the h2 for steps.

Reply:
Thanks for the correction, I misread the template